rllib/utils/actor_manager.py

```diff
@@ -237,11 +237,32 @@ def do_something(self):
     class _ActorState:
         """State of a single actor."""
 
-        # Num of outstanding async requests for this actor.
-        num_in_flight_async_requests: int = 0
+        # Num of outstanding async requests for this actor by tag.
+        num_in_flight_async_requests_by_tag: Dict[Optional[str], int] = field(
+            default_factory=dict
```

Comment:
Can we use here the `defaultdict` to save the `if` clause below?

Reply:
I'm not sure how. I tried and failed. Please take a stab at it. Curious to see how that would look.